Update readme and pkg description (#3)
DataDog/ssp-addon-datadog: README.md

# Datadog add-on for AWS SSP

> **This project is currently in Beta**

## Overview

> The Datadog SSP add-on allows deploying the Datadog Agent on Amazon EKS using the [ssp-amazon-eks](https://github.com/aws-quickstart/ssp-amazon-eks) [CDK](https://aws.amazon.com/cdk/) construct.

Reviewer: ```suggestion
The Datadog SSP add-on deploys the Datadog Agent on Amazon EKS using the [ssp-amazon-eks](https://github.com/aws-quickstart/ssp-amazon-eks) [CDK](https://aws.amazon.com/cdk/) construct.
```

const app = new cdk.App();

const addOns: Array<ssp.ClusterAddOn> = [
    new DatadogAddOn({
        // Kubernetes secret holding Datadog API key
        // The value should be set with the `api-key` key in the secret object.
        apiKeyExistingSecret: '<secret name>'
    })
];

const account = '<aws account id>'
const region = '<aws region>'
const props = { env: { account, region } }

new ssp.EksBlueprint(app, { id: '<eks cluster name>', addOns}, props)
```

## Add-on Options

| Option                  | Description                                         | Default                       |
|-------------------------|-----------------------------------------------------|-------------------------------|
| `apiKey`                | Your Datadog API key                                | ""                            |
| `appKey`                | Your Datadog APP key                                | ""                            |
| `apiKeyExistingSecret`  | Existing k8s Secret holding the API key             | ""                            |
| `appKeyExistingSecret`  | Existing k8s Secret holding the APP key             | ""                            |

> | `namespace`             | The namespace where to install the Datadog Agent    | "default"                     |

Reviewer: ```suggestion
| `namespace`             | Namespace where to install the Datadog Agent    | "default"                     |
```

> | `version`               | The version of the Datadog Helm chart               | "2.28.13"                     |

Reviewer: ```suggestion
| `version`               | Version of the Datadog Helm chart               | "2.28.13"                     |
```

| `release`               | The name of the Helm release                        | "datadog"                     |

> | `repository`            | The repository of the Helm chart                    | "https://helm.datadoghq.com"  |

Reviewer: ```suggestion
| `repository`            | Repository of the Helm chart                    | "https://helm.datadoghq.com"  |
```
